modules/dnn/test/test_misc.cpp

```diff
@@ -247,8 +247,7 @@ TEST(readNet, Regression)
     Net net = readNet(findDataFile("dnn/squeezenet_v1.1.prototxt"),
                       findDataFile("dnn/squeezenet_v1.1.caffemodel", false));
     EXPECT_FALSE(net.empty());
-    net = readNet(findDataFile("dnn/opencv_face_detector.caffemodel", false),
-                  findDataFile("dnn/opencv_face_detector.prototxt"));
```

Comment:
The new model is already covered by test in modules/objdetect/test/test_face.cpp. It does not make sense to infer it twice. The original model was not there and was covered by test to ensure the sample works well.

Reply:
OK, I will just remove this test.